_posts/2020-02-17-Combining-MongoDB-and-AWS-Lambda.md

```diff
@@ -0,0 +1,310 @@
+* Write down the security group associated with this vpc.
+Under security groups fetch the id of the security group that is associated with your vpc.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/7-security-group.png" width="70%" height="70%">
+</div>
+
+
+## The MongoDB side
+
+Setup the MongoDB cluster.
+**This has to be a dedicated cluster which means you'll need at least an M10.**
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/Setup-m10-cluster.png" width="70%" height="70%">
+</div>
+
+Wait till your cluster is set up.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/m10-is-setup.png" width="70%" height="70%">
+</div>
+
+In the Atlas UI navigate to `Security` -> `Network Access`.  
+Hit `+ new peering connection` and select AWS as cloud provider.  
+The below screen will pop up. Here you have to specify some configuration.
+* Account ID: your AWS account Id which you can find under the 'My Account' in the AWS console
+* VPC-id: Fill in the VPC-id that you copied from the vpc that you just created in AWS
+* VPC CIDR: specify the CIDR block that you used to configure your vpc with on AWS
+* region: the region where you created the AWS vpc
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/8-setup-peering-connection-atlas.png" width="70%" height="70%">
+</div>
+
+Hit `Instantiate peering` !
+
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/9-pending-peering-connection.png" width="70%" height="70%">
+</div>
+
+Notice that MongoDB created an `Atlas CIDR` which specifies the IP range in which your Atlas cluster will reside.  
+We are connecting the Atlas IP range with the IP range of our AWS VPC, hence VPC peering.
+
+The peering connection is now pending.   
+
+Go back to AWS.
+
+## The AWS side (again)
+In the VPC service of AWS go to `Peering Connections`.  
+You will a new peering request with stats `Pending Acceptance`.
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/10-aws-peering-connection-request.png" width="80%" height="80%">
+</div>
+
+Accept this peering request!
+
+Now you have to update your routing tables.
+AWS will also ask you `Do you want to update your routing tables` when you accept the peering request.  
+We will deploy our Lambda Functions in the public subnet of our VPC.
+So we want to modify the route table that is associated with that subnet.
+You can recognize that route table because it has an **explicit subnet association**.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/11-modify-route-table.png" width="100%" height="100%">
+</div>
+
+Click `edit routes` and add a route towards your Atlas cluster.  
```

Comment:
Edit which route table? In the image there are 3 different route tables

Reply:
Changed to Click `edit routes` when selecting the route table with an **explicit subnet association**

Reply:
Selecting the route table with an **explicit subnet association** and click `edit routes`.